src/main/java/frc/robot/subsystems/superstructure/arm/Arm.java

```diff
@@ -132,19 +131,15 @@ public void periodic() {
     }
 
     private boolean atPositionSetpoint() {
-        return setpoint.atSetpoint(inputs.leftPivotPositionRots, inputs.leftPivotVelocityRotsPerSec)
-                && setpoint.atSetpoint(inputs.rightPivotPositionRots, inputs.rightPivotVelocityRotsPerSec);
+        return setpoint.atSetpoint(inputs.pivotEncoderPositionRots, inputs.pivotEncoderVelocityRotsPerSec);
     }
 
     private boolean atPivotLowerLimit() {
-        return inputs.leftPivotPositionRots <= pivotSoftLowerLimit.pivotPositionRots
-                || inputs.rightPivotPositionRots <= pivotSoftLowerLimit.pivotPositionRots;
+        return inputs.pivotEncoderPositionRots <= pivotSoftLowerLimit.pivotPositionRots;
     }
 
     private boolean atPivotUpperLimit() {
-        return inputs.pivotUpperLimitSwitch
```

Comment:
Same here. LS need to stay removed, though, since LS isn't at upper limit of travel. That was a bug.